applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h

```diff
@@ -346,10 +347,11 @@ class DistanceModificationProcess : public Process
             {
                 if (rGeometry[itNode].GetSolutionStepValue(DISTANCE)<0.0)
                     inside++;
+                if (rGeometry[itNode].IsFixed(VELOCITY))
```

Comment:
You're right. Probably I committed a wrong file version. My bad. Thanks @RiccardoRossi .